Create Team Define Users

Tofudog opened the pull request:
This pull request mainly allows the chrome extension user to input two **Leetcode** usernames in the **Create Team Page**. Then, these two usernames will be the players of the game.

I also added `validateUserTest.js` to test whether a user is actually a **Leetcode** user.

What I need: `validateUser.js` to work.

Cyebukayire reviewed (approved):
Great work @Tofudog Thank you!

Tofudog commented:
**Update**

I also did the following:

-  Fixed the `isValid()` function
- Validated both players from the extension
- Solved an edge case where either of the two (valid) players have never submitted on Leetcode
- Added Unit Test Node JS file, which runs Jest

To use Jest, follow the steps:

1. `npm install --save-dev jest`
2. Then go to `package.json` and set `"test": "jest"` in **"scripts"**. You can just look at the `package.json` in my branch.
3. Each test file can be of the form `fileTest.test.js`

sunghokim128 reviewed (approved):
Seems legit!
